driver-core/src/main/java/com/datastax/driver/core/Row.java

```diff
@@ -20,8 +20,8 @@
 /**
  * A CQL Row returned in a {@link ResultSet}.
  * <p/>
- * The values of a CQL Row can be retrieve by either index (index starts at 0)
- * or name. When setting them by name, names follow the case insensitivity
+ * The values of a CQL Row can be retrieve by either index (index starts at zero)
+ * or name. When getting or setting them by name, names follow the case insensitivity
```

Comment:
This really should be "When getting them" only, as it is not possible to set anything in a Row.